Frameworks/CoreText/CTFramesetter.mm

```diff
@@ -16,96 +16,55 @@
 
 #import <CoreText/CTFrameSetter.h>
 #import <StubReturn.h>
+#import <algorithm>
 #import "CoreTextInternal.h"
 #import "CGPathInternal.h"
+#import "DWriteWrapper.h"
+
+using namespace std;
 
 @implementation _CTFrameSetter : NSObject
 - (void)dealloc {
     [super dealloc];
 }
 @end
 
-static id _createFrame(_CTFrameSetter* frameSetter, CGRect frameSize, CGSize* sizeOut, bool createFrame) {
+static _CTFrame* _createFrame(_CTFrameSetter* frameSetter, CGRect frameSize, CGSize* sizeOut, bool createFrame) {
     _CTFrame* ret = nil;
 
     if (createFrame) {
-        ret = [_CTFrame alloc];
+        ret = [_CTFrame new];
```

Comment:
elsewhere in this file too

---
In reply to: [77269955](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77269955) [](ancestors = 77269955)